ver/us/asm/data/542E0.data.s

```diff
@@ -2426,7 +2426,7 @@ glabel D_8008D294
 glabel D_8008D2B4
 .word 0x00000001, 0x0000003C, 0x000057E0
 
-glabel D_8008D2C0
+glabel dead_gItemTable
```

Comment:
this is a fake symbol, since the dead item table doesn't exist. The best thing to do here is remove the glabel and just append the data here to the end of the previous glabel. You can also remove D_8008D2C0 from undefined_syms and add it to dead_syms